test/runtime/lb.go

```diff
@@ -0,0 +1,269 @@
+package RunT
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/cilium/cilium/test/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var _ = Describe("RunLB", func() {
+
+	var initilized bool
+	var networkName string = "cilium-net"
+	var logger *log.Entry
+	var docker *helpers.Docker
+	var cilium *helpers.Cilium
+
+	initilize := func() {
+		if initilized == true {
+			return
+		}
+		logger = log.WithFields(log.Fields{"test": "RunLB"})
+		logger.Info("Starting")
+		docker, cilium = helpers.CreateNewRuntimeHelper("runtime", logger)
+		cilium.WaitUntilReady(100)
+		docker.NetworkCreate(networkName, "")
+		initilized = true
+	}
+
+	containers := func(mode string) {
+		var images map[string]string = map[string]string{
+			"httpd1": "cilium/demo-httpd",
+			"httpd2": "cilium/demo-httpd",
+			"httpd3": "cilium/demo-httpd",
+			"client": "tgraf/netperf",
+		}
+
+		switch mode {
+		case "create":
+			for k, v := range images {
+				docker.ContainerCreate(k, v, networkName, fmt.Sprintf("-l id.%s", k))
+			}
+		case "delete":
+			for k := range images {
+				docker.ContainerRm(k)
+			}
+		}
+	}
+
+	BeforeEach(func() {
+		initilize()
+		cilium.Exec("service delete --all")
+	}, 500)
+
+	AfterEach(func() {
+		containers("delete")
+	}, 500)
+
+	It("Service Simple tests", func() {
+
+		By("Creating a valid service")
+		result := cilium.ServiceAdd(1, "[::]:80", []string{"[::1]:90", "[::2]:91"}, 2)
+		Expect(result.Correct()).Should(BeTrue(),
+			"Service can't be added in cilium")
+
+		result = cilium.ServiceGet(1)
+		Expect(result.Correct()).Should(BeTrue(),
+			"Service can't be retrieved correctly")
+		Expect(result.Output()).Should(ContainSubstring("[::1]:90"), fmt.Sprintf(
+			"No service backends added correctly '%s'", result.Output()))
+
+		helpers.Sleep(5)
+		//FIXME: This need to be with Wait,Timeout
```

Comment:
If this can be punted, please create a GitHub issue and reference it here. See other comments in which I have requested this for more information.